.github/workflows/ci.yml

```diff
@@ -0,0 +1,82 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # 3.8, 3.9 do not work at the moment.
```

Comment:
Python 3.8 & 3.9 don't work? What's up with that? I'm using 3.8.5 on my machine, should I be using 3.7?

Reply:
I dunno. need to investigage. If you are using 3.8 successfully with GMT then maybe all is good.